### PR TITLE
Conversation: Fix Block Doesn't Load When Running WordPress 5.5

### LINKED ISSUE #18431: Conversation: Block Doesn't Load When Running WordPress 5.5
As reported in #18247 the Conversation block doesn't load at all when running WordPress 5.5:

Screen Shot on 2021-01-13 at 11:53:48

Console error:

```
TypeError: Object(...) is not a function
    ConversationEdit webpack:///./extensions/blocks/conversation/edit.js?:52
```

### PR DESCRIPTION
**Pull request by allilevine**
Fixes #18431

#### Changes proposed in this Pull Request:

* Remove useBlockProps to fix WordPress 5.5 compatibility issue

#### Jetpack product discussion

https://github.com/Automattic/jetpack/pull/18247#issuecomment-759651672

#### Does this pull request change what data or activity we track or use?

N/A

#### Testing instructions:

* Connect a site to Jetpack and enable beta blocks.
* Use the Jetpack Live Branches WP Downgrade option or the [WP Downgrade plugin](https://wordpress.org/plugins/wp-downgrade/) or `yarn docker:wp core update —version=5.5 —force` to downgrade to WordPress 5.5
* Add a Conversation block to a post.
* Inspect the block div and check that it has this class: `wp-block-jetpack-conversation`
* Check that the block is functional and has the same class in WordPress 5.6

For reference, this is what the div looked like before:
Screen Shot 2021-01-19 at 2 45 23 PM

#### Proposed changelog entry for your changes:

* No changelog entry needed; this is an update to a beta block.

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/conversation/edit.js [hunk 1]
```diff
@@ -29,9 +28,6 @@ const TRANSCRIPTION_TEMPLATE = [
 
 function ConversationEdit( { className, attributes, setAttributes } ) {
 	const { participants = [], showTimestamps, className: classNameAttr } = attributes;
-	const containerRef = useRef();
```
Comment (retrofox) [inline projects/plugins/jetpack/extensions/blocks/conversation/edit.js:32]: let's remove the `useRef` import some lines above.

File: projects/plugins/jetpack/extensions/blocks/conversation/edit.js [hunk 2]
```diff
@@ -2,12 +2,11 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { useEffect, useRef, useCallback } from '@wordpress/element';
+import { useEffect, useCallback } from '@wordpress/element';
```
Comment (retrofox) [inline projects/plugins/jetpack/extensions/blocks/conversation/edit.js:5]: 💯

File: projects/plugins/jetpack/extensions/blocks/conversation/edit.js (hunk 1 above)
Comment (allilevine) [reply projects/plugins/jetpack/extensions/blocks/conversation/edit.js:32]: Thanks! I removed it 👍

**Review by retrofox — APPROVED** [review]
LGTM. 👍

**Comment by retrofox** [comment]
> For reference, this is what the div looked like before:

The is nice, but unfortunately, the `useBlockProps` hook is not available in version5.5. I think we are going to switch to 5.6 as the minimum stable supported version by Jetpack. Then, we can start to use this hook for all blocks registered by it.

**Pull request merged**